client/js/keybinds.js

```diff
@@ -107,6 +107,19 @@ Mousetrap.bind(["alt+a"], function (e) {
 	return false;
 });
 
+// Show the help menu.
+let previousRoute;
+Mousetrap.bind(["command+/", "ctrl+/"], function (e) {
```

Comment:
This is hacky.
Is there a particular reason we can't use our router?
https://router.vuejs.org/guide/essentials/navigation.html#router-go-n

Reply:
line 111 that is, not sure why the other got included again

Reply:
Don't know how I managed to miss that, fixed!